### DISCUSSION TITLE
Abfrage Raster - Positionierung Pin

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#106] [Q&A]
Hallo Jürgen!

Es gibt möglicherweise einen Bug bei der Positionierung des Pins, wenn eine Abfrage auf einen Rasterlayer ausgeführt wird. 

Wir fragen im Imap (5.23.2801) die Werte eines Raster ab (z.B. Wassertiefen). 
Die Abfrage funktioniert, allerdings wird der Pin im Imap an einer falschen Stelle angezeigt (ca. 50 Meter entfernt). 
Beim Abfragen eines Vektors (z.B. Grundstücke) ist der Pin an der Stelle, an der geklickt wird. 

Im Beispiel wird eine WT (blauton) abgefragt und der Wert auch korrekt ausgelesen. Allerdings liegt der angezeigte Pin außerhalb des Wassertiefenlayers und nicht dort, wo ich geklickt habe. 

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/c3a3f62d-0bdb-48c2-b3f7-924f0017ecdc)

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/298ce438-0aa0-4861-8ee2-799145826220)

Lg, Andi

**Comment by gis-eni** [comment]
Hallo Andy,

das klinkt seltsam. Ein paar Infos brauche ich noch:

* Kann ich das nachvollziehen oder gibt es diese Abfrage nur in Intranet? Ich finde in eurer Fließgewässer Karte keine entsprechende Abfrage
* Fragst du hier mit dem "Identify Tool" ab?

Wenn Identify Tool, wäre hier fast auch der Dienst für mich interessant. Wenn man beim AGS etwas abfragt, kommt in der Regel auch eine Koordinate des Objektes zurück, dass gefunden wird. Das ist auch bei Raster Abfrage so. Vielleicht geht auf diesem Weg etwas schief :/

Bei unseren internen Diensten hab wird das Problem nicht. Wir haben allerdings auch alle Daten in der Regel im Karten Koordinaten System liegen (GK und nicht im WebMercator)
